tests/MacroTest.php

```diff
@@ -188,6 +188,30 @@ function ($a = null): int {
         $this->assertEquals('@return \stdClass|null rrrrrrr', $this->tagsToString($phpdoc, 'return'));
     }
 
+    public function testInitPhpDocParamsWithUnionTypes(): void
+    {
+        if (PHP_VERSION_ID < 80000) {
+            $this->markTestSkipped('This test requires PHP 8.0 or higher');
+            return;
```

Comment:
Please remove, not necessary -> `markTestSkipped` throws an exception

Reply:
Done!